docs/src/main/asciidoc/hibernate-orm-panache-guide.adoc

```diff
@@ -381,6 +381,29 @@ Make sure to wrap methods modifying your database (e.g. `entity.persist()`) with
 CDI bean method `@Transactional` will do that for you and make that method a transaction boundary. We recommend doing
 so at your application entry point boundaries like your REST endpoint controllers.
 
+JPA batches changes you make to entity and sends changes (it's called flush) at the end of the transaction or before a query.
```

Comment:
s/to entity/to your entities/

Reply:
fixed, I hesitated on this one when I copy/paste from Emmanuel's suggestion ... should have more convidence on y poor english ;)